tests/features/scheduler.feature

```diff
@@ -0,0 +1,47 @@
+@api @run
+Feature: Scheduler features
+  In order to moderate contents
+  As an administrator
+  I want to be able to schedule state transitions for contents
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+    And I change the variable "scheduler_publish_enable_page" to 1
+    And I change the variable "scheduler_unpublish_enable_page" to 1
```

Comment:
With a clean platform, this variable is already equals to 1. I think that this step is not useful.